samples/nrf_desktop/src/events/battery_event.c

```diff
@@ -46,7 +46,7 @@ EVENT_INFO_DEFINE(battery_state_event,
 		  profile_battery_state_event);
 
 
-EVENT_TYPE_DEFINE(battery_state_event, log_battery_state_event,
+EVENT_TYPE_DEFINE(battery_state_event, true, log_battery_state_event,
```

Comment:
please create a menu with events logging setting so this can be controlled by the user for each event

Reply:
done